frontend/src/components/Card.jsx

```diff
@@ -3,7 +3,7 @@ import "./Card.css";
 
 export default function Card({ card }) {
   return (
-    <article className={`card ${card.class}`}>
+    <article className={`card ${card.classe}`}>
```

Comment:
it would be necessary to use a function making it possible to standardize `card.classe` to respect the CSS class naming convention by removing capital letters and accent.